src/main/java/svydovets/BringApplication.java

```diff
@@ -0,0 +1,44 @@
+package svydovets;
+
+import org.apache.catalina.Context;
+import org.apache.catalina.LifecycleException;
+import org.apache.catalina.startup.Tomcat;
+import svydovets.core.context.ApplicationContext;
+import svydovets.web.DispatcherServlet;
+
+import java.io.File;
+
+public class BringApplication {
+
+    public static ApplicationContext run(Class<?> baseClass) {
+        try {
+           return configureTomcat(baseClass);
+        } catch (LifecycleException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static ApplicationContext configureTomcat(Class<?> baseClass) throws LifecycleException {
+        Tomcat tomcat = new Tomcat();
+        tomcat.setBaseDir("java.io.tmpdir");
+//        tomcat.setBaseDir("temp");
+        // Solution 1
+        tomcat.setPort(8080);
```

Comment:
I think it's better to move this info to the properties file. Let's do it in the next PR.